src/tools/cephfs/top/cephfs-top

```diff
@@ -203,8 +216,51 @@ class FSTop(object):
             # If the terminal do not support the visibility
             # requested it will raise an exception
             pass
-        self.run_display()
-
+        fs_map = self.get_fs_names()        
+        for filesystem in fs_map['filesystems']:
+            mdsmap = filesystem['mdsmap']
+            fs = mdsmap['fs_name']
+            fs_list.append(fs)
+        menu.append("Exit")
+        menu[0:0] = fs_list
+        self.run_all_display(fs_list)
+
+    def display_menu(self, stdscr, selected_row_idx):
+        stdscr.clear()
+        h, w = stdscr.getmaxyx()
+        for id, name in enumerate(menu):
```

Comment:
`id()` is a fundamental built-in in python, maybe changing `id` to `id_` would be better.

Reply:
Sure, changed to `index`